webapp/src/tutorial.tsx

```diff
@@ -528,6 +533,7 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
 
         const isRtl = pxt.Util.isUserLanguageRtl();
         return <div id="tutorialcard" className={`ui ${tutorialStepExpanded ? 'tutorialExpanded' : ''} ${tutorialReady ? 'tutorialReady' : ''} ${this.state.showSeeMore ? 'seemore' : ''}  ${!this.state.showHint ? 'showTooltip' : ''} ${hasHint ? 'hasHint' : ''}`} style={tutorialStepExpanded ? this.getExpandedCardStyle('height') : null} >
+            {hasHint && this.state.showHint && !unplugged && <div className='mask' onClick={this.closeHint}></div>}
```

Comment:
```suggestion
            {hasHint && this.state.showHint && !unplugged && <div className='mask' onClick={this.closeHint}></div>}
```

looks like this needs a role from travis